gcalcli

```diff
@@ -1377,12 +1388,22 @@ class gcalcli:
                     newStart, newEnd = GetTimeFromStr(val.strip(), length)
                     event['s'] = parse(newStart)
                     event['e'] = parse(newEnd)
-                    event['start'] = \
-                        {'dateTime': newStart,
-                         'timeZone': event['gcalcli_cal']['timeZone']}
-                    event['end'] = \
-                        {'dateTime': newEnd,
-                         'timeZone': event['gcalcli_cal']['timeZone']}
+
+                    if FLAGS.allday:
```

Comment:
Nope.  This is the EditEvent section and they DO exist.  The 'patch' function does not actually remove anything... it just adds/replaces, so you must explicitly set these to None or the function will break.